src/PageContent/Calendar/ContactContent.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import { AccountType, StringFormat } from '../../Enums';
+import './Content.css';
+import phone from '../../icons/phone.svg';
+
+class DescriptionContent extends Component {
+    constructor(props) {
+        super(props);
+        if (
+            this.props.delivery.donatingAgency &&
+            this.props.accountType === AccountType.DONATING_AGENCY_MEMBER
+        ) {
+            this.state = {
+                edit: false,
+                name: this.props.delivery.donatingAgency.contact.name,
+                phone: this.props.delivery.donatingAgency.contact.phone,
+                email: this.props.delivery.donatingAgency.contact.email
+            };
+        } else if (
+            this.props.delivery.receivingAgency &&
+            this.props.accountType === AccountType.RECEIVING_AGENCY
+        ) {
+            this.state = {
+                edit: false,
+                name: this.props.delivery.receivingAgency.contact.name,
+                phone: this.props.delivery.receivingAgency.contact.phone,
+                email: this.props.delivery.receivingAgency.contact.email
+            };
+        } else {
+            this.state = {
+                edit: false
+            };
+        }
+
+        this.edit = this.edit.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+    }
+    edit() {
+        this.setState({
+            edit: true
+        });
+    }
+
+    handleChange(e) {
+        e.preventDefault();
+        this.setState({
+            name: e.target.name.value,
+            phone: e.target.phone.value,
+            edit: false
+        });
```

Comment:
Read the comment below about 2 different edit modes for DA vs RA first.
There needs to be 2 modes here too
```
let contact = {};
if (this.props.accountType === AccountType.DONATING_AGENCY_MEMBER) {
    contact = this.state.memberList[e.target.contact.value];
} else if (this.props.accountType === AccountType.RECEIVING_AGENCY) {
    contact = {
        name: e.target.name.value,
        phone: e.target.phone.value,
        email: e.target.email.value
    };
}
// backend TODO: write to db
this.setState({contact: contact});
```